backend/infrahub/git/tasks.py

```diff
@@ -6,18 +6,50 @@
 from infrahub.core.protocols import CoreRepository
 from infrahub.core.registry import registry
 from infrahub.exceptions import RepositoryError
+from infrahub.git.repository import InfrahubRepository, get_initialized_repo
 from infrahub.services import services
 
 from ..log import get_logger
 from ..tasks.artifact import define_artifact
 from ..workflows.catalogue import REQUEST_ARTIFACT_DEFINITION_GENERATE, REQUEST_ARTIFACT_GENERATE
 from ..workflows.utils import add_branch_tag
-from .models import RequestArtifactDefinitionGenerate, RequestArtifactGenerate
-from .repository import InfrahubRepository, get_initialized_repo
+from .models import GitRepositoryAdd, RequestArtifactDefinitionGenerate, RequestArtifactGenerate
 
 log = get_logger()
 
 
+@flow(name="git-repository-add-read-write")
+async def add_git_repository(model: GitRepositoryAdd) -> None:
+    log.info(
```

Comment:
I think we can remove this log it's redundant with what we are getting from prefect directly